doc/OnlineDocs/getting_started/index.rst

```diff
@@ -1,9 +1,58 @@
 Getting Started
 ===============
 
-Installation, pyomo command, a simple example, etc
+Installation
+------------
 
-.. doctest::
+Pyomo currently supports the following versions of Python:
+
+* CPython: 2.7, 3.5, 3.6
+
+
+Using CONDA
+~~~~~~~~~~~
+
+We recommend installation with *conda*, which is included with the Anaconda
+distribution of Python.  If you have a different Python distribution, then
+you can install *miniconda* using *pip*:
+
+.. bash::
+    pip install miniconda
+
+You can install Pyomo in your system Python installation
+by executing the following in a shell:
+
+.. bash::
+    conda install -c conda-forge pyomo
+
+Pyomo also has conditional dependencies on a variety of third-party Python packages.  These can also be installed with conda:
+
+.. bash::
+    conda install -c conda-forge pyomo.extras
+
+Optimization solvers are not installed with Pyomo, but some open source optimization solvers can be installed with conda as well:
+
+.. bash::
+    conda install -c conda-forge pyomo.solvers
```

Comment:
This should be `conda install -c conda-forge ipopt coin-cbc glpk`

Reply:
Done.